teacher.rb

```diff
@@ -0,0 +1,12 @@
+require '/person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(name, age, parent_permission)
```

Comment:
- [ ] You are passing up the `parent_permission` value but did not define it in the parameters. Kindly define the `parent_permission` on the **`initialize`** method parameter.

Reply:
Yes, sorry, I realized that my mistake was passing the parameter.